api/api_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const testdbname = "db.test"
+const testdbname = "/tmp/db.test"
```

Comment:
What is the purpose of changing the path?

Not every OS (windows for example) has /tmp dir (or not every user has access to /tmp dir)